Postman Test Suite/Ed-Fi ODS-API Composite Base Test Suite.postman_collection.json

```diff
@@ -729,6 +729,74 @@
 										"description": "Scenario: Composites support GetByKey pattern requests\r\n    Given the subject of the request is \"Staff\" where\r\n    | ParameterName | Value  | Type |\r\n    | StaffUniqueId | 207288 | Key  |    \r\n    When a GET (all) request is submitted to the \"StaffFilter\" composite with the query string filter\r\n    Then the response model should only contain the records with exact matches by id"
 									},
 									"response": []
+								},
+								{
+									"name": "GetByKey pattern using a nonexistent key",
```

Comment:
This test covers a nonexisting key scenario but is hitting an endpoint that does not exist. Either rename the test to match what is validating or use an existing endpoint

Reply:
The endpoint does exist, it's hitting the [Get endpoint](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS/blob/main/Application/EdFi.Ods.Features/Controllers/CompositeResourceController.cs#L83) for Composites.
It covers [this line of code](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS/blob/d697debfa7ce4cb0428c868fb7c8795fdf80b13b/Application/EdFi.Ods.Features/Composites/Infrastructure/CompositeResourceResponseProvider.cs#L125).

Reply:
Oh, you are right; this is a composite; it is the same 404 exception when the endpoint does not exist.